.eslintrc.js

```diff
@@ -1,45 +1,27 @@
 module.exports = {
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/nodejs',
-  ],
-  plugins: [
-    'json',
-  ],
-  parserOptions: {
-    ecmaVersion: 2018,
-  },
+  root: true,
+
+  extends: ['@metamask/eslint-config'],
+
   overrides: [
     {
       files: ['*.ts'],
-      extends: [
-        '@metamask/eslint-config/config/typescript',
-      ],
+      extends: ['@metamask/eslint-config-typescript'],
     },
+
     {
-      files: [
-        '*.js',
-        '*.json',
-      ],
+      files: ['*.js'],
       parserOptions: {
         sourceType: 'script',
```

Comment:
Do we need this? Our ESLint Node.js config has this comment:

>// The recommended Node.js plugin config sets the correct `sourceType` per the
>// `type` field of the local package.json file, so we don't set that here.
That implies to me that we don't need to set this explicitly.

Reply:
You're right. Good call. Removed accordingly in 6abe7426028bc6ff37c88945a494bdd11ecc38a2.

Reply:
This was probably taken from the module template? Looks like the same mistake is made there as well

Reply:
Yup! Okay, I'll make a change there as well.